consensus/scheme/rolldpos/fsm.go

```diff
@@ -409,17 +426,17 @@ func (m *cFSM) Start(c context.Context) error {
 					logger.Debug().Msg("timeoutEvt is stale")
 					continue
 				}
+				if evt.Height() < m.ctx.round.height {
+					logger.Debug().Msg("message of a previous round")
+					continue
+				}
+				if evt.Height() > m.ctx.round.height {
```

Comment:
I wonder if we should requeue in this case. When we starts a new round, round is either not set, or is still the last round cache.